examples/DC2_examples/DC2_gt_profiles.ipynb

```diff
@@ -396,7 +396,7 @@
    "outputs": [],
    "source": [
     "dc2_cosmo = extragalactic_cat.cosmology\n",
-    "cosmo = clmm.Cosmology(H0 = dc2_cosmo.H0, Omega_dm0 = dc2_cosmo.Om0-dc2_cosmo.Ob0, Omega_b0 = dc2_cosmo.Ob0)\n",
+    "cosmo = clmm.Cosmology(H0 = dc2_cosmo.H0.value, Omega_dm0 = dc2_cosmo.Om0-dc2_cosmo.Ob0, Omega_b0 = dc2_cosmo.Ob0)\n",
```

Comment:
@hsinfan1996 - I just ran the notebook from master, i.e. without the change you made to `dc2_cosmo.H0` and it ran fine. So, I'm a bit puzzled as to why that change is needed... You ran at NERSC?

Reply:
I ran at NERSC and on my laptop, and they all gave errors. I am thinking if this problem is unique to me too.

Reply:
I tested it on NERSC and I found I had to use `dc2_cosmo.H0.value`, because otherwise it would give an error related to ccl.
I used `desc-python` rather than `conda-clmmenv` to import gcr catalogs, and I adjusted CLMM imports by adding 
```
import sys
sys.path.append('../../')
```

Reply:
Thanks @shenmingfu. So I finally managed to reproduced the error and it happens with the CCL and NC backends. The cluster-toolkit backend is fine because it uses Astropy cosmology object which can handle `quantities`, but not the two other backends. So yes, we need `dc2_cosmo.H0.value` for it to work with all cosmo backends.

Thanks for spotting the issue @hsinfan1996!

Reply:
oh great, thank you. @combet @shenmingfu